docs/tutorial/using-native-node-modules.md

```diff
@@ -58,10 +58,14 @@ where to download headers and which version to use:
 
 ```bash
 $ cd /path-to-module/
-$ HOME=~/.electron-gyp node-gyp rebuild --target=0.29.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
+$ HOME=~/.electron-gyp node-gyp rebuild --target=0.36.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
+#On Windows Try this
+cd /path-to-module/
+X:\path-to-module>node-gyp rebuild --target=0.36.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
```

Comment:
We shouldn't give instructions assuming people use Bash/Cygwin - the people who use this are usually experts already and can translate. The CMD version is:

```sh
SET HOME="%USERPROFILE%\.electron-gyp"
node-gyp rebuild --target=0.36.1 --arch=x64 --dist-url=https://atom.io/download/atom-shell
```